doc/source/serve/tutorial.rst

```diff
@@ -0,0 +1,89 @@
+==============
+Basic Tutorial
```

Comment:
Not sure I like this name, maybe we can call it "End-to-end Tutorial" ?

Reply:
walkthrough?